src/main/java/at/fh/joanneum/irfc/persistence/entiy/EventInfoEntity.java

```diff
@@ -0,0 +1,39 @@
+package at.fh.joanneum.irfc.persistence.entiy;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import javax.persistence.*;
+import java.util.List;
+
+/**
+ * @author gregor.wakonig@edu.fh-joanneum.at
+ **/
+@Entity
+@NoArgsConstructor
+@Getter
+@Setter
+@Table(name = "event_info")
+public class EventInfoEntity {
+
+    @Id
+    @SequenceGenerator(
+            name = "event_info_id_seq",
+            sequenceName = "event_info_id_seq",
+            allocationSize = 1
+    )
+    @GeneratedValue(
+            strategy = GenerationType.SEQUENCE,
+            generator = "event_info_id_seq"
+    )
+    @Column(name = "event_info_id")
+    private Long eventInfoId;
+
+    @Column(name = "event_info_text")
+    private String infoText;
+
+    @OneToMany
+    @JoinColumn(name = "fk_image", nullable = false)
```

Comment:
@PhilipMoser not sure how those relations should work in Hibernate (can you take a quick look over the relations if this is correct) :)

Reply:
the one to many side should be mappedBy and not annotated with a join column.
you can check out the EventLocationEntity and how it is mapped to the EventEntity and
the explanation in this post https://examples.javacodegeeks.com/java-development/enterprise-java/hibernate/hibernate-foreign-key-example/ is helpful in my opinion. :)